md/getting-started-tutorial.md

```diff
@@ -175,16 +175,16 @@ To define the business data model:
 3. Click on the temporary name, and change it to _TravelRequest_.
 4. Select the _TravelRequest_ business object type, and add its attributes. This table shows the attributes and their types:
 
-  | Variable  | Data type | Mandatory |
-  | --------- | --------- | -------- |
-  | userId  | Long  | yes  |
-  | departureDate  | Date  | yes  |
-  | numberOfNights  | Integer  | yes  |
-  | hotelNeeded  | Boolean  | yes  |
-  | destination  | String  | yes  |
-  | reason  | String  | yes  |
-  | status  | String  | yes  |
-  | refusalReason  | String  | no  |
+| Variable  | Data type | Mandatory |
+| --------- | --------- | -------- |
+| userId  | Long  | yes  |
+| departureDate  | Date  | yes  |
+| numberOfNights  | Integer  | yes  |
+| hotelNeeded  | Boolean  | yes  |
+| destination  | String  | yes  |
+| reason  | String  | yes  |
+| status  | String  | yes  |
+| refusalReason  | String  | no  |
```

Comment:
The extra two spaces does not align the table with the ordered list items but lead to an issue in columns content display on our web site. Easy fix is to remove the two spaces.